OTRAssets/Strings/OTRStrings.h

```diff
@@ -10,6 +10,8 @@ FOUNDATION_EXPORT NSString* ABOUT_VERSION_STRING();
 FOUNDATION_EXPORT NSString* ACCOUNTS_STRING();
 /** "Account", The string describing a buddy's account */
 FOUNDATION_EXPORT NSString* ACCOUNT_STRING();
+/** "Active", segmented control value for active friends */
+FOUNDATION_EXPORT NSString* ACTIVE_BUDDIES_STRING();
```

Comment:
We already have a string for "Active" called `ACTIVE_STRING()`, should we just use that? Perhaps this would be translated differently in other languages. One is implying "Last user interaction" and the other is "Not archived".